src/config/icons.ts

```diff
@@ -0,0 +1,47 @@
+import { ARBITRUM, AVALANCHE } from "config/chains";
+import arbitrum from "img/ic_arbitrum_24.svg";
+import avalanche from "img/ic_avalanche_24.svg";
+import gmxIcon from "img/ic_gmx_40.svg";
+import glpIcon from "img/ic_glp_40.svg";
+import gmxArbitrum from "img/ic_gmx_arbitrum.svg";
+import gmxAvax from "img/ic_gmx_avax.svg";
+import glpArbitrum from "img/ic_glp_arbitrum.svg";
+import glpAvax from "img/ic_glp_avax.svg";
+
+type Icons = {
+  [ARBITRUM]: {
+    network: string;
+    gmx: string;
+    glp: string;
+  };
+  [AVALANCHE]: {
+    network: string;
+    gmx: string;
+    glp: string;
+  };
+  gmx: string;
+  glp: string;
```

Comment:
why do we need separate gmx/glp icons per network?

Reply:
This helps to differentiate between the active network. For better idea you can look at the icon below:

<img width="66" alt="Screenshot 2023-01-10 at 2 38 02 PM" src="https://user-images.githubusercontent.com/73279781/211508738-5018d0ff-f6d2-496f-aaa5-61331d1c01b8.png">